src/app/api/shared/interfaceUtils.spec.ts

```diff
@@ -50,9 +50,9 @@ describe('browserSettingsApi', () => {
 });
 
 describe('getDeviceInterface', () => {
-    it('throws exception when mode is not electron', () => {
+    it('does not throw exception when mode is not electron', () => {
```

Comment:
I think this should probably be reframed as an assertion like 'call factory x' or removed outright.